gui/vm/utils.py

```diff
@@ -10,3 +14,35 @@ def vm_enabled():
             return True
         else:
             return False
+
+@locked(DW_PROGRESS)
+def dump_download_progress(data):
+    with open(DW_PROGRESS, 'w') as dw_f:
+            details_msg = 'Lets start the magic...'
+            progress_n = data.get('progress').get('percent')
+            if progress_n:
+                if progress_n <= 10:
+                    details_msg = 'Start to download a prebuilt container image.'
+                elif progress_n > 10 and progress_n <= 50:
+                    details_msg = 'We are almost there....'
```

Comment:
'Almost done...'

(Although this is a little misleading and could be shown for a long time if the user is on a slow connection.)